### PR TITLE
Remove remaining Firefox Android flags + old Firefox desktop flags

### LINKED ISSUE #9679: Firefox for Android 79+ doesn't support flags
Like #4753 and #7354, Firefox for Android 79+ (Fenix) doesn't appear to ship access to flags / preferences in general releases. That is, `about: config` doesn't do anything in Firefox for Android.

We ought to close any open support statements with flags with a `"version_removed": "79"`, or report `"version_added": false` for such features added at 79 or later.

This would not appear to apply to Firefox for Android version 69 and earlier (i.e., Fennec).

### PR DESCRIPTION
**Pull request by queengooborg**
This PR removes all of the remaining Firefox Android flag data from BCD that isn't already covered by other PRs, as well as similar Firefox desktop flag data.  Fixes #9679.  Blocked by #15698.

### CODE REVIEW

File: html/global_attributes.json
Comment (foolip) [inline html/global_attributes.json:1135]: Do you know what explains the big difference to the desktop data above?

File: html/manifest/icons.json
Comment (foolip) [inline html/manifest/icons.json:22]: Desktop is null above. Should it be?

File: http/headers/feature-policy.json
Comment (foolip) [inline http/headers/feature-policy.json:503]: The behavior described here sounds indistinguishable from no support before Firefox 80, assuming just the `allowfullscreen` attribute is enough and adding the `allow` attribute doesn't change anything. Can you check where this note came from and if it makes any sense?

Maybe split this out into a separate PR if it needs tweaking beyond the script-made changes.

File: http/headers/feature-policy.json
Comment (foolip) [inline http/headers/feature-policy.json:21]: Sounds to me like no support for the header! Should this be false with notes instead? If so, can you fix in a separate PR?

File: http/headers/feature-policy.json
Comment (foolip) [inline http/headers/feature-policy.json:406]: No Android?

File: http/headers/feature-policy.json
Comment (foolip) [inline http/headers/feature-policy.json:1507]: For a header, no effect means no support, this note should be removed in favor of the correct version then.

File: http/headers/origin.json
Comment (foolip) [inline http/headers/origin.json:38]: Here's an "ambiguous gap" that the guideline talks about.

File: javascript/builtins/Atomics.json
Comment (foolip) [inline javascript/builtins/Atomics.json:50]: How about moving this note to the entry that remains? You did this in one case below, let's have the same everywhere! 😀

And let's rephrase the note to talk about removal rather than "disabled by default".

File: javascript/builtins/DataView.json
Comment (foolip) [inline javascript/builtins/DataView.json:199]: Same comment about the note phrasing here.

**Review by foolip — COMMENTED** [review]
Some things here will need multiple round trips I think, can you separate out anything that needs manual fixes to keep this PR simple to review? Grouping everything that has to do with SAB being disabled in a single PR would be great.

File: html/global_attributes.json
Comment (queengooborg) [reply html/global_attributes.json:1135]: It appears that this was added by #6782.  The [linked bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1631681) shows there was an intentional offset between enabling the feature on desktop and mobile.

File: html/manifest/icons.json
Comment (queengooborg) [reply html/manifest/icons.json:22]: Desktop support is probably `false` as well, but I'm not totally confident on that.  I'd like to hold off digging into it until we commence our no-nulls project for HTML though?

File: http/headers/feature-policy.json
Comment (queengooborg) [reply http/headers/feature-policy.json:503]: It appears the notes came from #6493, as well as #12548.

File: http/headers/feature-policy.json
Comment (queengooborg) [reply http/headers/feature-policy.json:21]: I opened #15987 because this seems like something we'd have to dig deeper into first.

File: http/headers/feature-policy.json
Comment (queengooborg) [reply http/headers/feature-policy.json:406]: Seems that none of the mobile browsers are marked as supported.  I guess that makes sense.

File: http/headers/origin.json
Comment (queengooborg) [reply http/headers/origin.json:38]: I mean, technically, we're not _only_ leaving behind a `version_added: true`; we _do_ have `version_added: 70` up above, so this _is_ technically within the guidelines?  😝 

(I definitely would want to try and find a version number for this, but I'm not yet certain how to find version numbers for HTTP headers and I wouldn't want our flag removal to be blocked on it...)

File: http/headers/feature-policy.json
Comment (foolip) [inline http/headers/feature-policy.json:173]: I'm not sure about removing the "dom.security.featurePolicy.header.enabled" flag from the parent but leaving it here with a bumped version. Can you back out these changes and leave them for https://github.com/mdn/browser-compat-data/issues/15987?

File: http/headers/origin.json
Comment (foolip) [reply http/headers/origin.json:38]: OK, let's leave this a mystery for the future :)

**Review by foolip — APPROVED** [review]
OK to land with Feature-Policy changes split out.

File: http/headers/feature-policy.json
Comment (foolip) [reply http/headers/feature-policy.json:503]: OK, maybe this can be simplified then.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
@queengooborg there are more flags that need to be removed, CI is failing.

File: http/headers/feature-policy.json
Comment (queengooborg) [reply http/headers/feature-policy.json:503]: I opened #16537 for us to take a look into this at a later time!

File: http/headers/feature-policy.json
Comment (queengooborg) [reply http/headers/feature-policy.json:1507]: #15987 was opened to look further into these notes.  I agree that these notes imply `version_added: false`, but would not like to have them block this PR and would love to keep the scope of this PR as small as possible.

**Comment by queengooborg** [comment]
CI is now passing since the separated VR changes have been merged, and all of the questions regarding the questionable notes have issues opened for us to look into after this is merged.  This is ready for re-review, I'd say!

**Comment by queengooborg** [comment]
Oh I just realized that an approval has already been given!

**Pull request merged**
